courses/machine_learning/deepdive2/image_classification/labs/4_tpu_training.ipynb

```diff
@@ -0,0 +1,275 @@
+{
```

Comment:
<ul><li>"the same before," -&gt; "the same as before,</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/853/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>